pkg/image/validation/kubernetes_test.go

```diff
@@ -140,33 +140,38 @@ func TestIsKubernetesDefined(t *testing.T) {
 	assert.False(t, result)
 }
 
-func TestValidateNodes(t *testing.T) {
+func TestValidateNetwork(t *testing.T) {
+	var validV4Network = validNetwork
+
 	tests := map[string]struct {
 		K8s                    image.Kubernetes
 		ExpectedFailedMessages []string
 	}{
-		`valid`: {
+		`valid v4 VIP`: {
 			K8s: image.Kubernetes{
-				Network: validNetwork,
-				Nodes: []image.Node{
-					{
-						Hostname: "agent1",
-						Type:     image.KubernetesNodeTypeAgent,
-					},
-					{
-						Hostname:    "server",
-						Type:        image.KubernetesNodeTypeServer,
-						Initialiser: true,
-					},
+				Network: validV4Network,
+			},
+		},
+		`valid v6 VIP`: {
+			K8s: image.Kubernetes{
+				Network: image.Network{
+					APIHost: "host.com",
+					APIVIP:  "FEC0::1",
 				},
 			},
 		},
-		`no nodes`: {
+		`valid single node - no network config`: {
 			K8s: image.Kubernetes{
-				Nodes: []image.Node{},
+				Network: image.Network{},
+				Nodes: []image.Node{
+					{
+						Hostname: "host",
+						Type:     image.KubernetesNodeTypeServer,
+					},
+				},
 			},
 		},
-		`with nodes - no network config`: {
+		`invalid multi-node - no network config`: {
```

Comment:
I have moved this test from Node validation to Network validation as I also have removed the node test from there too. I think it makes more sense here, but let me know what you think.